apps/expo-go/src/navigation/defaultNavigationOptions.tsx

```diff
@@ -19,7 +19,8 @@ export default (theme: ColorTheme): StackNavigationOptions => {
       color: theme === 'dark' ? darkTheme.text.default : lightTheme.text.default,
     },
     headerTintColor: theme === 'dark' ? darkTheme.icon.default : lightTheme.icon.default,
-    headerBackTitleVisible: false,
+    headerBackButtonDisplayMode: 'minimal',
+    headerBackImage: () => <></>,
```

Comment:
@satya164 is there no equivalent to `headerBackTitleVisible` in react-navigation 7? this feels like a non-obvious way to accomplish the equivalent

Reply:
@lukmccall - was it intentional to add `headerBackImage: () => <></>` here? it seems like the behavior we had before hid the title, but not the back image.